src/Systems.jl

```diff
@@ -10,7 +10,7 @@ struct SpinSystem{N}
     _dipoles    :: Array{Vec3, 4}                   # Holds dipole moments: Axes are [Basis, CellA, CellB, CellC]
     _coherents  :: Array{CVec{N}, 4}                # Coherent states
     site_infos  :: Vector{SiteInfo}                 # Characterization of each basis site
-    S           :: Array{ComplexF64, 3}    
+    S           :: Array{ComplexF64, 3}             # TODO -- generate on demand instead
```

Comment:
I guess you found this broke something when removed?

Reply:
Good catch. This was easy to remove. See the most recent commit, [9341612](https://github.com/SunnySuite/Sunny.jl/pull/48/commits/93416129ff041daaefede434acd38c4fa661a0e9)